.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 apiary.html
+.idea
```

Comment:
Don't think this belongs here though. 
Different editors will create similar folders, meaning we would need to keep updating this file to account for external tools. 

Add this to your `.gitignore_global` and keep this file specific to the repo internal usage.